src/vs/workbench/contrib/webview/browser/resourceLoading.ts

```diff
@@ -119,23 +118,22 @@ function getResourceToLoad(
 	requestUri: URI,
 	roots: ReadonlyArray<URI>,
 	remoteAuthority: string | undefined,
-	useRootAuthority: boolean | undefined
 ): URI | undefined {
 	for (const root of roots) {
 		if (containsResource(root, requestUri)) {
-			return normalizeResourcePath(requestUri, remoteAuthority, useRootAuthority ? root.authority : undefined);
+			return normalizeResourcePath(requestUri, remoteAuthority);
 		}
 	}
 
 	return undefined;
 }
 
-function normalizeResourcePath(resource: URI, remoteAuthority: string | undefined, useRemoteAuthority: string | undefined): URI {
+function normalizeResourcePath(resource: URI, remoteAuthority: string | undefined): URI {
 	// If the uri was from a remote authority, make we go to the remote to load it
-	if (useRemoteAuthority || (remoteAuthority && resource.scheme === Schemas.file)) {
+	if (remoteAuthority && resource.scheme === Schemas.file) {
```

Comment:
I think that ideally, asWebviewUri would create the correct URIs with remote info, and we wouldn't need this

Reply:
Good idea! I'm going to merge this current change as-is since I was able to test it but will investigate rewriting the uri on the extension host side